android/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for Android enables you to integrate your SPA with our services.
```

Comment:
Not sure that "SPA" applies to native apps does it? (I might be misunderstanding what that means)

Reply:
@brando-dill change this to something like:
`The Ping SDK for Android enables you to integrate your Android application with Ping services.`

Reply:
Done and done.